<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportWidget</class>
 <widget class="QGroupBox" name="ExportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="formatLabel">
        <property name="text">
         <string>Format </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="formatComboBox">
        <item>
         <property name="text">
          <string>PNG</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>JPEG</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TGA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BMP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HDR</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="jpegQualityLabel">
        <property name="text">
         <string>JPEG Quality </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="SliderSpinBox" name="jpegQualityWidget" native="true"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fileNameLabel">
        <property name="text">
         <string>File Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>Path</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="FileSelector" name="fileSelector" native="true"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="toneMappingLabel">
        <property name="text">
         <string>Tone Mapping</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="toneMappingComboBox">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reinhard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Exposure</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="exposureLabel">
        <property name="text">
         <string>Exposure</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="exposureSpinBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="fileNameLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="suffixLabel">
           <property name="text">
            <string>.png</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exportButton">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderSpinBox</class>
   <extends>QWidget</extends>
   <header>SliderSpinBox.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileSelector</class>
   <extends>QWidget</extends>
   <header>FileSelector.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
